src/vaadin-menu-bar-sub-menu-mixin.html

```diff
@@ -89,6 +89,16 @@
       }
     }
 
+    __fireItemSelected(value) {
+      this.dispatchEvent(
+        new CustomEvent('item-selected', {
+          detail: {value},
+          bubbles: true,
```

Comment:
Is there a need for the event to bubble & be composed?

Reply:
Good question. Probably not, I think we can drop them for now and add later if needed.